src/functions/on-call/create-on-call-data-from-json/handler.ts

```diff
@@ -0,0 +1,88 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+import { middyfy } from "@libs/lambda";
+import type { OnCallImportEntry, OnCallImportError } from "src/database/models/oncall";
+import * as jwt from 'jsonwebtoken';
+
+/**
+ * Lambda method for importing on-call data from JSON
+ *
+ * Expects body to be an array of objects with "Week" (number) and "Person" (string) properties
+ * and a query parameter "year" (number)
+ */
+export const onCallImportFromJsonHandler = async (event) => {
+  const token = event.headers?.Authorization?.split(' ')[1];
+  const decodedJWT = jwt.decode(token);
+  const isAdmin = decodedJWT?.realm_access?.roles?.includes("admin") || false;
+  if (!isAdmin) {
+    return {
+      statusCode: 403,
+      body: "Forbidden: admin access required",
+    };
+  }
```

Comment:
Can you not use the reusable utility function that does this that Juuso set up, it's in:
/src/libs/auth-utils.ts